components/Header.js

```diff
@@ -11,7 +12,8 @@ export default function Header({
       <div className="flex  justify-around p-4 md:p-8 items-center px-4 lg:px-48 w-full ">
         <div>
           <h1 className="text-5xl font-bold pb-3">{title}</h1>
-          <h2>{subtitle}</h2>
+          <h1 className="text-5xl font-bold pb-3">{title2}</h1>
+          <h2 className="pb-6">{subtitle}</h2>
           <h2> {subtitle2}</h2>
```

Comment:
make the subtitle text to a  bigger size also the header needs a lot more top and bottom padding.